app/models/subscription.rb

```diff
@@ -0,0 +1,6 @@
+class Subscription < ApplicationRecord
+  belongs_to :contact
+  belongs_to :list
+
+  enum status: %i[pending complete]
```

Comment:
related to comment elsewhere, maybe useful to have `%i[pending_create created pending_destroy]` (no `:destroyed` because the record no longer exists) -- "pending" and "complete" don't really convey what is complete and what is pending.